packages/logging_cloudwatch/amplify_logging_cloudwatch/lib/src/queued_item_store/index_db/indexed_db_adapter.dart

```diff
@@ -138,15 +138,14 @@ class IndexedDbAdapter implements QueuedItemStore {
     await request.future;
 
     for (final elem in request.result) {
-      final value = elem as Map<String, dynamic>;
-      final id = value['id'] as int;
-      final itemValue = value['value'] as String;
-      final timestamp = value['timestamp'] as String;
+      final value = elem as Object;
```

Comment:
This code appears to be a duplicate of `getCount` - can we please refactor the code to avoid duplication?